### PR TITLE
library logic

### PR DESCRIPTION
**Pull request by Tresor11**
on this branch we implemented the following features:
-add book 
-remove book 
-change book status

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off",
+      "no-unused-vars":"off",
```
Comment (bellom) [inline .eslintrc.json]: You made some changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

File: library.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  this.pages = pages;
+  this.status = status;
+}
+
+function render() {
+  let tableRows = '';
+
+  myLibrary.forEach(book => {
+    tableRows += `<tr>
+<th scope='row' class='text-center'><span class='badge badge-primary'>${
```
Comment (bellom) [inline library.js]: Kindly fix all indentation in this file.

File: library.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+function Book(title, author, pages, status) {
+  this.title = title;
+  this.id = myLibrary.length;
+  this.author = author;
+  this.pages = pages;
+  this.status = status;
+}
+
+function render() {
```
Comment (bellom) [inline library.js]: Let have our DOM manipulation task separated from application logic
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/

File: library.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+function addBookToLibrary() {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const pages = document.getElementById('pages').value;
+  const status = document.getElementById('status').checked;
+  const newBook = new Book(title, author, pages, status);
+  if (title === '') {
+    window.alert('please provide a title');
```
Comment (bellom) [inline library.js]: It is not encouraged to use `alert` in production.
I guess you use `required` on the input tag, with that you don't have to change the linter's config file.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
### 1. Kindly check out my comments below

### 2. [Optional]
It would be nice to have the `id` of the books in the correct order.
image

example: 
image


**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: .eslintrc.json (hunk 1 above)
Comment (Tresor11) [reply .eslintrc.json]: the linter doesn't allow unused functions however in this case the function add is called by the  DOM event

File: library.js (hunk 2 above)
Comment (Tresor11) [reply library.js]: that was indentation was recommended by the linter

Comment (bellom) [reply library.js]: Looks scattered at my end,  yours?

File: .eslintrc.json (hunk 1 above)
Comment (bellom) [reply .eslintrc.json]: In this case, we could use inline come;
`/* eslint-disable no-unused-vars */` 🤔

Comment (Tresor11) [reply .eslintrc.json]: thank you for the info we'll be using that :smiley:

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :warning: 
Hi there team :hand: 
You did a nice work implementing the requested changes :clap: 
However, it seems you forgot one.

- [x] It's best practice to let the DOM manipulation tasks outside our app logic. You can achieve that by moving the render function to another file and then use export and import to use it. Those concepts were taught in the [ES06 lesson warm up](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/37953388/).
Screenshot from 2020-03-13 10-29-53

Please kindly submit for code review again after making this change
Happy coding :computer: 
**Sergio Torres**

**Comment by Tresor11** [comment]
There is only one Js file because we are not using webpack on this project and thus import/export will not work
Thank you @Torres-ssf  for the precision

**Review by shloch — APPROVED** [review]
## Hello @Tresor11  , @ampaire 
### well done...........all points corrected

accomplished

### you can merge this

**Pull request merged**